### PR TITLE
few updates

### PR DESCRIPTION
**Pull request by JefferyMitchell**
Wording and alignment

### CODE REVIEW

**Comment by PRMerger14** [comment]
@JefferyMitchell : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/ready/enterprise-scale/transition.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ To understand which move strategy you should use, we will go through examples of
 
 ## Subscription move 
 
-The common use cases for moving subscriptions are primarily to 1) organize subscriptions into management groups, and 2) transfer subscriptions to a new Azure Active Directory tenant. We will focus on moving subscriptions to management groups in this section as moving to a new tenant is mainly for [transferring billing ownership](https://docs.microsoft.com/azure/cost-management-billing/manage/billing-subscription-transfer). 
+The common use cases for moving subscriptions are to organize subscriptions into management groups or when transfering subscriptions to a new Azure Active Directory tenant. Subscription moves for enterprise-scale focuses on moving subscriptions to management groups, when moving a subscription to a new tenant this is mainly for [transferring billing ownership](https://docs.microsoft.com/azure/cost-management-billing/manage/billing-subscription-transfer).
```
Comment (davidkpiano) [inline docs/ready/enterprise-scale/transition.md:33]: > Subscription moves for enterprise-scale focuses on moving subscriptions to management groups, when moving a subscription to a new tenant this is mainly for

This reads a little weird

**Comment by PRMerger13** [comment]
@BrianBlanchard : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@BrianBlanchard : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by BrianBlanchard** [comment]
#sign-off

**Pull request merged**